# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openscenegraph/hotfix-no-ffmpeg.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- OpenSceneGraph-OpenSceneGraph-3.6.5/CMakeLists.txt.vanilla	2024-06-01 20:13:29.567428515 +0200
+++ OpenSceneGraph-OpenSceneGraph-3.6.5/CMakeLists.txt	2024-06-01 20:13:36.275428479 +0200
@@ -752,7 +752,7 @@
     FIND_PACKAGE(CURL)
     FIND_PACKAGE(LibVNCServer)
     FIND_PACKAGE(DCMTK)
-    FIND_PACKAGE(FFmpeg)
+    #FIND_PACKAGE(FFmpeg)
     FIND_PACKAGE(GStreamer COMPONENTS app pbutils)
     FIND_PACKAGE(GLIB COMPONENTS gobject)
     FIND_PACKAGE(DirectShow)
